tests/test_normalize_fields.py

```diff
@@ -0,0 +1,125 @@
+from concurrent.futures import ThreadPoolExecutor
+import unittest
+from datetime import datetime, timezone
+
+import preql
+
+from data_diff.database import BigQuery, MySQL, Snowflake, connect_to_uri, Oracle, DEFAULT_PRECISION
+from data_diff.sql import Select
+from data_diff import database as db
+
+from .common import CONN_STRINGS
+
+
+DATE_TYPES = {
```

Comment:
Do we need this test suite at all when we have the test_types one?

Reply:
It tests timezones. test_types doesn't at the moment.

Reply:
Ok... I think in a follow-up for time-zone support, we should remove this test suite and just have the other one only. Simplifies a lot.

Reply:
If it covers all the cases, then sure.